static-analysis/teammates-eslint-es6.yml

```diff
@@ -20,6 +20,15 @@ rules:
     # enforce a maximum line length
     max-len: ["error", 125]
 
+    # disallow continue statements
+    no-continue: "off"
+
+    # disallow mixes of different operators
+    no-mixed-operators: "off"
```

Comment:
Though mixing operators can be abused, also agree that doing so sometimes improves readibility as long as only common precedence rules are relied on, e.g * vs +. Reviews'll catch places where its not so obvious.